docs/api/router-dashboard.md

```diff
@@ -1703,3 +1703,25 @@ Enables remote access for debugging and user use, only usable from the local mes
 - Sample Call:
 
 `curl http://192.168.10.1:4877/remote_access/true`
+
+## /wyre_enabled
+
+Returns a boolean indicating if the Wyre payment dialog should be enabled in the dashboard according to the oracle
+
+- URL: `<rita ip>:<rita_dashboard_port>/wyre_enabled`
```

Comment:
Please make this a localization endpoint which returns an object that contains wyre enabled